app/grandchallenge/reader_studies/models.py

```diff
@@ -484,7 +484,7 @@ def remove_reader(self, user):
     @property
     def help_text(self) -> str:
         """The cleaned help text from the markdown sources"""
-        return md2html(self.help_text_markdown, link_blank_target=True)
+        return md2page_html(self.help_text_markdown)
```

Comment:
Maybe this goes back to which filter am I supposed to use, but why do we need to embed youtube videos in help text?

Reply:
I got confused by this name as well. But it is actually the text that gets shown on the reader study Information page.